src/components/People.js

```diff
@@ -0,0 +1,71 @@
+import "../App.css";
+
+import React, { Component } from 'react'
+
+class People extends Component {
+    constructor(){
+        super()
+        this.state={
+            characterData:[],
+            userInput: ""
+        }
+
+    }
+
+    fetchCharacters=()=>{
+        fetch("https://ghibliapi.herokuapp.com/people")
+        .then(res=> res.json())
+        .then((data)=>{
+            this.setState({
+                characterData: data
+            })
+        })
+    }
+
+    componentDidMount(){
+        this.fetchCharacters()
+    }
+
+    handleUserInput=(event)=>{
+        this.setState({
+            userInput: event.target.value
+        })
+    }
+    handleSubmit=(event)=>{
+        event.preventDefault()
+     
+    }
```

Comment:
```suggestion
    handleSubmit=(event)=>{
        event.preventDefault()
     this.setState({
     selectedCharacter: this.state.userInput
     })
    }
```
Now that you have a way to keep track of the `selectedCharacter`, you can update it in your `handleSubmit` function. When the user submits a search, the screen will change to show either:
1) the character's info, if a successful search was made
OR
2) "Not Found", if the search was unsuccessful